terraform/aws/scenarios/omnibus-external-elasticsearch/variables.tf

```diff
@@ -0,0 +1,70 @@
+#########################################################################
+# AWS
+#########################################################################
+variable "aws_profile" {
+  type        = "string"
+  description = "Name of the AWS profile used for authentication (e.g. chef-engineering)."
+  default     = "chef-engineering"
+}
+
+variable "aws_region" {
+  type        = "string"
+  description = "Name of the AWS region to create instances in (e.g. us-west-2)."
+  default     = "us-west-1"
```

Comment:
we usually use us-west-2 region. can we default to that?

Reply:
`us-west-1` is the default in terraform because `us-west-2` doesn't have any room to support creating any more VPC's and the terraform code is written first for a dev to run from the command line.

The `integration_test` buildkite pipeline overrides that default and points it to `us-west-2`.